basic-configuration.ttl

```diff
@@ -406,22 +409,25 @@ stage:GeoAppearance a elmo:Representation;
 	elmo:endpoint <http://linkedgeodata.org/sparql>;
 	elmo:appearance elmo:GeoSelectAppearance;
 	elmo:queryForm elmo:GeoForm;
-	elmo:query '''
-		Prefix lgdo:<http://linkedgeodata.org/ontology/>
-		Prefix geom:<http://geovocab.org/geometry#>
-		Prefix ogc: <http://www.opengis.net/ont/geosparql#>
-		prefix geo: <http://www.w3.org/2003/01/geo/wgs84_pos#>
-		construct {
-		#	?h ogc:asWKT ?hgv
-			?s geo:geometry ?g
-		}
-		WHERE {
-		?s a lgdo:Hospital ;
-			rdfs:label ?l ;
-			geom:geometry [ ogc:asWKT ?g]
+	elmo:query stage:HopitalQuery
+.
+	stage:HospitalQuery a elmo:query;
```

Comment:
Dit zou moeten zijn: stage:HospitalQuery a elmo:Query (hoofdletter: elmo:query is een property, elmo:Query is de klasse)